examples/006_filter_composite_data_example.py

```diff
@@ -4,9 +4,15 @@
 Filter result data by different criteria
 ----------------------------------------
 
-This example show how data filtering can be used for custom postprocessing of
+This example shows how data filtering can be used for custom postprocessing of
 layered composites. You can filter strains and stresses by material, layer, or
-analysis ply. The example filters data by layer, spot, and node, as well as material
+analysis ply. Filtering by analysis ply is implemented on the server side and
+exposed with the function :func:`.get_ply_wise_data`. In this case the data is
```

Comment:
```suggestion
exposed with the :func:`.get_ply_wise_data` function. In this case, the data is
```
Google dev style guide says to put the noun describing a tagged entity after the entity.